server/ai_process.go

```diff
@@ -1043,10 +1044,30 @@ func submitLiveVideoToVideo(ctx context.Context, params aiRequestParams, sess *A
 		if err != nil {
 			return nil, fmt.Errorf("control pub url - %w", err)
 		}
+		controlStr := control.String()
+
 		clog.V(common.VERBOSE).Infof(ctx, "pub %s sub %s control %s", pub, sub, control)
 		startTricklePublish(pub, params)
 		startTrickleSubscribe(sub, params)
 		startControlPublish(control, params)
+
+		//Switch subscribe and publish urls
+		req.SubscribeUrl = pub.String() // publish url
+		req.PublishUrl = sub.String() // subscriber url
+		req.ControlUrl = &controlStr
+		
+		// Send another request to the orchestrator to send the video to the live pipeline
```

Comment:
I am a little confused why this is needed, can you describe what is happening at a high level maybe in the PR description?

Reply:
@eliteprox added on PR desc now!